optuna/trial/_trial.py

```diff
@@ -603,6 +616,13 @@ def should_prune(self) -> bool:
             configured pruning algorithm. Otherwise, the trial should continue.
         """
 
+        if len(self.study.directions) > 1:
+            _logger.warning(
+                "When the problem is multi-objective optimization, "
+                "Trial.should_prune always returns `False`."
```

Comment:
What are the guidelines for putting backticks? 🤔

Reply:
Hmm... I'm not sure about how to use backticks in logging messages. I'll not use any backticks in logging messages.

Reply:
Backticks are more like markups in my opinion so omitting them for error messages sounds reasonable to me.